modules/search-service-impl/src/main/java/org/opencastproject/search/impl/solr/SolrIndexManager.java

```diff
@@ -213,6 +213,49 @@ public void clear() throws SolrServerException {
     }
   }
 
+  /**
+   * Removes the entry with the given <code>id</code> from the database.
+   *
+   * @param seriesid
+   *          identifier of the series or episode to delete
+   * @param deletionDate
+   *          the deletion date
+   * @throws SolrServerException
+   *           if an errors occurs while talking to solr
+   */
+  public boolean deleteSeries(String seriesid, Date deletionDate) throws SolrServerException {
```

Comment:
That's basically the same method as the one below, isn't it? I think it would be better to put that into a private method called deleteElement() or something, and just use it in the public method instead of duplicating the code.